pyanaconda/bootloader.py

```diff
@@ -98,6 +98,35 @@ def _is_on_ibft(device):
 
     return all(getattr(disk, "ibft", False) for disk in device.disks)
 
+def _get_petitboot_version():
+    """
+    get the version of the petitboot loader that OF expresses as:
+        v1.6.1-pd8c7a0a
+    in the form:
+        ('v1.6.1-pd8c7a0a', (1, 6, 1), 'pd8c7a0a')
+    """
+    path = "/sys/firmware/devicetree/base/ibm,firmware-versions/petitboot"
+    if not os.access(path, os.F_OK):
+        raise FileNotFoundError
+    for line in open(path).readlines():
+        if line.startswith('v'):
+            line = line[1:]
+        (components, digest) = line.split('-')
+        components = [int(x) for x in components.split('.')]
+        return (line, tuple(components), digest)
+
+    return (None, (), None)
+
+def _version_ge(version, goal):
```

Comment:
Same here.

Reply:
@poncovka you are right, I've pushed a new version that also removes those two.